katcp/object_proxies.py

```diff
@@ -1,20 +1,15 @@
-# Copyright 2014 National Research Foundation (South African Radio Astronomy Observatory)
```

Comment:
Any reason as to why you remove the license header?

Reply:
The file was sourced from somewhere else, and has its own CP right header